apps/web/lib/emails/templates/attendee-scheduled-email.ts

```diff
@@ -6,11 +6,12 @@ import utc from "dayjs/plugin/utc";
 import { createEvent, DateArray } from "ics";
 import nodemailer from "nodemailer";
 
+import { serverConfig } from "@calcom/lib/serverConfig";
+
```

Comment:
I moved this from web/lib to packages/lib so website app can call it too. all the email templates are updated